### ISSUE TITLE
Förslag på teman

### ISSUE THREAD

**Issue by redhog** [ohanssen/IT-Etikk#2]
* Konflikten mellan yttrandefrihet och upphovsrätt
* Kryptering och konflikten stat/medborgare

* Machine learning / AI
  * Personvärn / integritet
  * Diskriminering
  * Automatisk censur / "mjuk" censur (jfr facebooks prioritering för feeden)

* Dual-use - hur en programmvara kan komma att missbrukas
* Syftesglidning - hur ett företag eller institution avsiktligt långsamt ändrar den ursprungliga planen för hur något skall användas. Sluttande-plan-argumentet.

* Förhållande mellan utvecklare och arbetsgivare
  * NDA:er
  * Attribution vs företagets upphovsrätt
  * Kan man säga nej till vissa typer av uppdrag, kan/skall man göra det redan i en interviewsituation (se t.ex. https://medium.freecodecamp.com/the-code-im-still-ashamed-of-e4c021dff55e#.a2tvtxf2q )

**Comment by ohanssen** [comment]
Ja, dette er jo viktige spørsmål som bør tas opp i en eventuell lærebok.

**Comment by ohanssen** [comment]
I dag kommer vi vel ikke utenom Shoshana Zuboff - suveillance capitalism.
